release-common.nix

```diff
@@ -57,7 +57,7 @@ rec {
       git
       mercurial
     ]
-    ++ lib.optional stdenv.isLinux [libseccomp pkgs.utillinux]
+    ++ lib.optional stdenv.isLinux [libseccomp utillinux]
```

Comment:
Also should use lib.optionals (notice the "s") to avoid placing a nested list

https://github.com/NixOS/nixpkgs/blob/master/lib/lists.nix#L200-L222